x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/add_prebuilt_rules_table/use_add_prebuilt_rules_table_columns.tsx

```diff
@@ -132,11 +134,11 @@ export const useAddPrebuiltRulesTableColumns = (): TableColumn[] => {
   const hasCRUDPermissions = hasUserCRUDPermission(canUserCRUD);
   const [showRelatedIntegrations] = useUiSetting$<boolean>(SHOW_RELATED_INTEGRATIONS_SETTING);
   const {
-    state: { loadingRules, isRefetching, isUpgradingSecurityPackages },
+    state: { loadingRules, isRefetching, isUpgradingSecurityPackages, isInstallingAllRules },
     actions: { installOneRule },
   } = useAddPrebuiltRulesTableContext();
 
-  const isDisabled = isRefetching || isUpgradingSecurityPackages;
+  const isDisabled = isRefetching || isUpgradingSecurityPackages || isInstallingAllRules;
```

Comment:
Same here.

Reply:
Ok, thank you. I will keep that in mind for some refactoring PR touching this place.